LEGO1/act3.h

```diff
@@ -25,6 +25,10 @@ class Act3 : public LegoWorld
     return !strcmp(name, Act3::ClassName()) || LegoWorld::IsA(name);
   }
 
+  inline void SetUnkown420c(MxEntity *p_entity) { m_unk420c = p_entity; }
+  protected:
+    undefined m_unkf8[0x4114];
```

Comment:
Same here, `DECOMP_SIZE_ASSERT` would be great. Helps prevent some accidental changes to the structure.